tls/s2n_x509_validator.c

```diff
@@ -347,6 +347,14 @@ s2n_cert_validation_code s2n_x509_validator_validate_cert_chain(struct s2n_x509_
             }
         }
 
+        /* certificate extensions is a field in TLS 1.3 */
+        if (conn->actual_protocol_version == S2N_TLS13) {
+            uint16_t certificate_extensions_length = 0;
```

Comment:
Q: Why is the extensions length a `uint16_t` value?

Reply:
from the rfc, it has a 16 bit range `Extension extensions<0..2^16-1>;` 
https://tools.ietf.org/html/rfc8446#section-4.4.2

Reply:
Thank-you, so this is the holding the extensions and not just the length of it. In that case maybe call it the `certificate_extensions` instead?

Reply:
`certificate_extensions_length` does hold the length. the length is required to read the actual extensions (which we are skipping over) the next line.